cmd/gateway-gcs-layer.go

```diff
@@ -197,10 +218,28 @@ func (l *gcsGateway) MakeBucket(bucket string) error {
 func (l *gcsGateway) MakeBucketWithLocation(bucket, location string) error {
 	bkt := l.client.Bucket(bucket)
 
-	// TODO: location
-	_ = location
+	// this will map s3 regions to google multi regions
+	if v, ok := map[string]string{
```

Comment:
Yes, but we need to make exceptions then for ca- and sa-. Thought this was cleaner.